components/02-molecules/alert/_yds-alert.scss

```diff
@@ -4,6 +4,7 @@
 @use '../../01-atoms/atoms';
 
 $alert-themes: map.deep-get(sass-tokens.$tokens, color, alert);
+$alert-fade-speed: var(--animation-speed-slow);
 
 .alert {
   --size-icon: 1.25rem;
```

Comment:
Right after these custom properties are set, we can use the `animate` mixin to make our "visual" changes. I like how you did opacity, but I'm also going to animate the `max-height` to prevent the little bit of "overlapping text" that can happen the way it is now. So I'd just add this:

```
  @include tokens.animate($duration: $alert-fade-speed);
```

And then after the `color` definition, let's default alerts to

```
opacity: 1;
max-height: 7rem; // This is just a random number I chose. If we go too much higher, it'll make the animation speed seem fast... and it should be plenty large enough to accommodate the amount of text they want to allow in the alerts.
```